testing/vcs/test_vcs_init_open_sizing.py

```diff
@@ -46,12 +46,6 @@ def test_canvas_size(c, size, via):
     info = c.canvasinfo()
     w, h = size
 
-    # Make sure size fits on screen bounds
-    screen_w, screen_h = c.backend.renWin.GetScreenSize()
```

Comment:
@doutriaux1 if you remember @dlonie pushed a fix for this and now it should work on all systems.